src/component/project/Project/Project.tsx

```diff
@@ -80,26 +86,84 @@ const Project = () => {
+                                show={
+                                    <StyledDiv>
+                                        <StyledTitle data-loading>
+                                            Description:&nbsp;
+                                        </StyledTitle>
+                                        <StyledText data-loading>
+                                            {project.description}
+                                        </StyledText>
+                                    </StyledDiv>
+                                }
+                            />
+                            <StyledDiv>
+                                <StyledTitle data-loading>
+                                    projectId:&nbsp;
+                                </StyledTitle>
+                                <StyledText data-loading>
+                                    {projectId}
+                                </StyledText>
+                            </StyledDiv>
                         </div>
-                        <PermissionIconButton
-                            permission={UPDATE_PROJECT}
-                            projectId={projectId}
-                            sx={{ visibility: isOss() ? 'hidden' : 'visible' }}
-                            onClick={() =>
-                                navigate(`/projects/${projectId}/edit`)
-                            }
-                            tooltipProps={{ title: 'Edit project' }}
-                            data-loading
-                        >
-                            <Edit />
-                        </PermissionIconButton>
+                        <StyledDiv>
+                            <PermissionIconButton
+                                permission={UPDATE_PROJECT}
+                                projectId={projectId}
+                                sx={{
+                                    visibility: isOss() ? 'hidden' : 'visible',
+                                }}
+                                onClick={() =>
+                                    navigate(`/projects/${projectId}/edit`)
+                                }
+                                tooltipProps={{ title: 'Edit project' }}
+                                data-loading
+                            >
+                                <Edit />
+                            </PermissionIconButton>
+                            <PermissionIconButton
+                                permission={DELETE_PROJECT}
+                                projectId={projectId}
+                                sx={{
+                                    visibility: isOss() ? 'hidden' : 'visible',
+                                }}
+                                onClick={() => {
+                                    setShowDelDialog(true);
+                                }}
+                                tooltipProps={{ title: 'Remove project' }}
```

Comment:
I noticed we are using `remove` in some places and `delete` in others. 
Do you know if there is a reasoning or guideline we can follow?

Reply:
I think we used to have `remove` alot, but now slowly moving to `delete` on everywhere. Changed this to delete also.